OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -157,7 +157,7 @@ public void UnReserve()
 			}
 		}
 
-		public void Killed(Actor self, AttackInfo e)
+		public void Disposing(Actor self)
 		{
 			UnReserve();
```

Comment:
Isn't disposing meant for cleaning up held references, memory and things like that? If so, I wouldn't expect gameplay logic in a `Disposing` method. In other words, what's wrong with keeping this in the `Killed` method?

(I just picked this line as an example, same thing applies to others.)